test/test-basic-properties.js

```diff
@@ -250,6 +250,19 @@ it('inheritance is considered in type checking', () => {
   assert(el.objectProperty === array, 'property was set');
 });
 
+it('numeric properties deserialize "" (empty) to "NaN"', () => {
```

Comment:
The spec locks down our new opinions. All of `''`, `' '`, `'    '` should be coerced to `NaN`. Note that no other tests were depending on the prior `Number('') >> 0` behavior.